### ISSUE TITLE
Exception thrown: 'System.NotImplementedException' for  MaxPooling2D

### ISSUE THREAD

**Issue by User2587** [SciSharp/Keras.NET#46]
I'm getting the below error:

onnxmltools
Warning! onnxmltools is not installed to export to onnx
tensorflowjs
Warning! tensorflowjs is not installed. Required to convert keras model to tensorflow js format
2019-10-05 21:46:27.040654: I C:\Users\User\Source\Repos\tensorflow\tensorflow\core\platform\cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
Exception thrown: 'System.NotImplementedException' in MyProj.exe
An unhandled exception of type 'System.ArgumentException' occurred in System.Drawing.dll
Parameter is not valid.

for this code:
        model.Add(new MaxPooling2D(pool_size: (2, 2).ToTuple(), strides: (2, 2).ToTuple(), padding: "same"));

**Comment by User2587** [comment]
The same code was working in python environment.

**Comment by deepakkumar1984** [comment]
Fixed with latest version
